demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -1879,3 +1893,78 @@ def execute_tests(self):
             raise
         finally:
             self.build_context.logging_module.execute_logs()
+
+
+def replace_external_playbook_configuration(client: DefaultApi, external_playbook_configuration: dict,
+                                            logger_module: logging.Logger = logging.getLogger('demisto-sdk')):
+    """ takes external configuration of shape {"playbookID": "Isolate Endpoint - Generic V2",
+                                               "input_parameters":{"Endpoint_hostname": ["simple", "test"]}
+        and changes the specified playbook configuration to the mentioned one.
+        Returns (Whether the Playbook changed, The values to restore, the path to use when restoring)
+        Only to be used with server version 6.2 and above. """
+    if not external_playbook_configuration:
+        logger_module.info("External Playbook Configuration not provided, skipping re-configuration.")
+        return False, {}, ''
+    server_version = get_demisto_version(client)
+
+    if LooseVersion(server_version.base_version) < LooseVersion('6.2.0'):  # type: ignore
+        logger_module.info("External Playbook not supported in versions previous to 6.2.0, skipping re-configuration.")
```

Comment:
@JasBeilin Did you communicate this to @idovandijk ?

Reply:
Yes, I explained that the api call for checking the inputs is unavailable before 6.2.